WoWsShipBuilder.Core/Utility/GameDataHelper.cs

```diff
@@ -69,4 +69,98 @@ public static string GetCategoryString<T>()
             _ => throw new InvalidOperationException(),
         };
     }
+
+    public static char ClassToUrlIndex(ShipClass shipClass)
```

Comment:
Rename this. This maps a class to a letter, not to an url. Keep the name of the method closer to what it actually does.
The same applies for the other methods